### ISSUE TITLE
H2O equilibrium calc with direct HITEMP file  raises spectroscopic constants not found  

### ISSUE THREAD

**Issue by erwanp** [radis/radis#297]
### 🐛 Describe the bug

Reported by [周周zz](https://twitter.com/zz99583643/status/1408739988439666689)  on Twitter

### 🖥 Steps To Reproduce

```python

from astropy import units as u
from radis import calc_spectrum
s = calc_spectrum(wavenum_min = 7500 / u.cm,
                  wavenum_max = 7600 / u.cm,
                  molecule='H2O',
                  isotope='1,2,3',
                  pressure=1.01325 * u.bar,
                  mole_fraction=0.58,
                  path_length=6.8 * u.cm,
                  verbose=3,
                  Tgas=3000,   
                  databank=r'\PATH_TO_\01_7500-8000_HITEMP2010.par',
                  )
s.plot('radiance_noslit', nfig='same')
```

Raises : 

```
KeyError: "H2O is not defined in molecules with built-in spectroscopic constants. Choose one of: ['CO', 'CO2']"
```

Which is needed only for nonequilibrum calculation (which is not required here)

**Comment by erwanp** [comment]
Error reproduced on 0.9.29. 

It is already fixed on 0.9.30 (developer-version) , probably because of https://github.com/radis/radis/pull/252

Same code on current develop branch : 

![image](https://user-images.githubusercontent.com/16088743/123511027-35660080-d67f-11eb-8c59-544a60b9fce4.png)

**Comment by erwanp** [comment]
Already fixed by @anandxkumar   in #252  , closing.
